BrainPortal/app/views/messages/new.html.erb

```diff
@@ -44,10 +44,10 @@
      </p>
 
      <p>
-     To users of project: <%= group_select(:group_id,
+     To users of project: <%= group_select(:destination_group_id,
```

Comment:
No changes needed in this file, this group_id is again the group of people to send a message to, not an invitation group ID.